### PR TITLE
Hackathon/tailings

### PR DESCRIPTION
**Pull request by mmumby**
# MAIN
- Updated Tailing Storage Facility form to include more information
Screen Shot 2021-04-27 at 4 38 26 PM
- Exposed "Tailings" tab at all times
- Updated Tailings view
Screen Shot 2021-04-27 at 4 39 07 PM
Screen Shot 2021-04-27 at 4 40 59 PM
- Added a Map to the tailings tab to show all TSF that have lat/long set
Screen Shot 2021-04-27 at 4 41 43 PM
- Map can show or Hide layers including TSF layer
Screen Shot 2021-04-27 at 4 41 59 PM
- Added a popup to the tailings pin on map
Screen Shot 2021-04-27 at 4 42 31 PM

# Other
- Added a count to all tab sections on a Mine Record that contains a table
Screen Shot 2021-04-27 at 4 39 53 PM
Screen Shot 2021-04-27 at 4 40 19 PM
Screen Shot 2021-04-27 at 4 40 35 PM

### CODE REVIEW

File: services/core-web/src/components/Forms/AddTailingsForm.js
Comment (mmumby) [inline services/core-web/src/components/Forms/AddTailingsForm.js:34]: Ant design select components do not allow `false` as a value, had to cast to a string in order to use the select with booleans.

File: services/core-web/src/components/mine/Summary/MineSummary.js
Comment (mmumby) [inline services/core-web/src/components/mine/Summary/MineSummary.js:98]: This fixes the issue where the link on the map (on the MineHeader) cannot be clicked because the row covers it, I moved the row down slightly so the link is now clickable.

File: services/core-web/src/components/Forms/AddTailingsForm.js
Comment (exzizt) [inline services/core-web/src/components/Forms/AddTailingsForm.js:110]: Not required?

**Review by exzizt — COMMENTED** [review]
Awaiting some more discussed changes before approval

File: services/core-web/src/components/Forms/AddTailingsForm.js
Comment (mmumby) [reply services/core-web/src/components/Forms/AddTailingsForm.js]: Correct, not required, and this has been deleted.

**Review by exzizt — APPROVED** [review]

**Review by exzizt — APPROVED** [review]

**Pull request merged**
